agent/consul/server_test.go

```diff
@@ -177,7 +177,7 @@ func testServerConfig(t *testing.T) (string, *Config) {
 
 	// TODO (slackpad) - We should be able to run all tests w/o this, but it
 	// looks like several depend on it.
-	config.RPCHoldTimeout = 5 * time.Second
+	config.RPCHoldTimeout = 10 * time.Second
```

Comment:
Is this necessary for tests to pass?

Reply:
Yes, I don't recall exactly (it's been a few months) but according to the commit message I left in cb4de4dc7e07b008f594ed5cc0b1701fbafcf990, the 5-second `RPCHoldTimeout` was causing some timeouts when running many tests concurrently.